frontend/components/WaitingPages/HostWaiting.tsx

```diff
@@ -15,30 +16,61 @@ const inter = Inter({
 });
 
 interface hostProps {
-	hostId: string;
+	gameId: string;
 }
 
-export default function HostWaiting({ hostId }: hostProps) {
-	const [timeLimit, setTimeLimit] = useState<string | null>(null);
-	const [maxPlayers, setMaxPlayers] = useState<string | null>(null);
+const colors = [
+	'bg-red-300',
+	'bg-green-300',
+	'bg-purple-350',
+	'bg-blue-300',
+	'bg-purple-300',
+];
+
+export default function HostWaiting({ gameId }: hostProps) {
+	const [timeLimit, setTimeLimit] = useState<number>(15);
+	const [maxPlayers, setMaxPlayers] = useState<number>(5);
 	const playerList = useContext(playerListContext) as playerListContextType;
+	const router = useRouter();
+
+	// timeLimit Modifiers for every change
+	function incrementTime() {
```

Comment:
In my most recent PR for the quiz initialization page, I think I created a very similar component for the timer -/+ that you might be able to use here if you merge with my branch. Mine does faster increments/decrements when the buttons are held after a certain threshold of time.